src/Microsoft.DotNet.Build.Tasks/EncryptedConfigNuGetRestore.cs

```diff
@@ -35,7 +35,6 @@ public override bool Execute()
                 GlobalPackagesFolder = PackagesDir,
 
                 Sources = new List<string>(),
-                FallbackSources = new List<string>(),
```

Comment:
Is this a relevant change?

Reply:
This property is not there in "4.3.0" a change made in project.json